pandas/io/parsers/readers.py

```diff
@@ -442,7 +440,6 @@
 }
 
 _deprecated_defaults: dict[str, Any] = {"error_bad_lines": None, "warn_bad_lines": None}
-_deprecated_args: set[str] = {"error_bad_lines", "warn_bad_lines"}
```

Comment:
looks like both of these are present in the parsers.pyi file but not in parsers.pyx.  Does one of those need to be updated?